### PR TITLE
Optional change longitude and latitude per query

### PR DESCRIPTION
**Pull request by mistadave**
# changes

* choose latitude and longitude in query
* add docker-compose for dev
* add webpack-dev config
* add uglifyjs-webpack-plugin
* updated readme
* update to version 1.0.1

# Query view

Added the latitude and longitude to the query. By default the values from the Datasource setting will be used. Can manually be changed on each query.
Screenshot from 2020-03-03 16-59-58

# Supported Grafana version

I testet it from grafana 3.x.x up to 6.x.x. I could't it only get working with Grafana >= 5.0.0. The earlier versions got an error with the database proxy.

### CODE REVIEW

File: dist/partials/query.editor.html [hunk 1]
```diff
@@ -9,16 +9,24 @@
...
           allow-custom="true"
           lookup-text="true"
           get-options="ctrl.getOptions($query)"
           on-change="ctrl.refresh()">
         </gf-form-dropdown>
       </div>
     </div>
-    <div class="gf-form gf-form--grow">
-      <div class="gf-form-label gf-form-label--grow"></div>
+  <div class="gf-form-inline">
```
Comment (andig) [inline dist/partials/query.editor.html]: Indentation looks wrong?

File: src/partials/query.editor.html [hunk 2]
```diff
@@ -9,16 +9,24 @@
...
           allow-custom="true"
           lookup-text="true"
           get-options="ctrl.getOptions($query)"
           on-change="ctrl.refresh()">
         </gf-form-dropdown>
       </div>
     </div>
-    <div class="gf-form gf-form--grow">
-      <div class="gf-form-label gf-form-label--grow"></div>
+  <div class="gf-form-inline">
```
Comment (andig) [inline src/partials/query.editor.html]: Indents?

Comment (mistadave) [reply src/partials/query.editor.html]: your right, i fixed it  with my next commit

**Comment by andig** [comment]
Published 1.0.1 but do not feel like going through the pain of getting this release registered with Grafana- feel free to open the plugin PR if you want.

**Comment by mistadave** [comment]
I'll try. Maybe it gets through :smiley:

**Pull request merged**